packaging/test-systemd.sh

```
@@ -184,5 +184,130 @@ EOF"
+  first=$(migration_state)
+  [ "${first%%:*}" -gt 0 ] 2>/dev/null \
+    && ok "swap: ce applied its migrations ($first)" \
+    || bad "swap: no migrations recorded after ce started ($first)"
+
+  # A row only this deployment could have written, so "the data is still there" is a claim about
+  # content rather than about the tables existing.
+  docker exec "$PG" psql -U yorishiro -d yorishiro \
+    -c "CREATE TABLE IF NOT EXISTS swap_marker (note text); INSERT INTO swap_marker VALUES ('written-under-ce');" >/dev/null 2>&1
+
+  install_edition ee \
+    && ok "swap: ee starts on the database ce created" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)